### ISSUE TITLE
ci: make official list of Narwhals gifs to use in PR reviews

### ISSUE THREAD

**Issue by MarcoGorelli** [narwhals-dev/narwhals#629]
Currently I just use: https://chriskuehl.github.io/shipit/

But people have been very creative with their gifs

We should probably have an official list, and then, when we make a "maintainers' guide", we instruct people to please use one from the list when submitting reviews

dead serious here

**Comment by anopsy** [comment]
Yes, let's extend that list with some dolphin, orcas and maybe whales gifs (as narwhals gifs are almost non existent). I'll try to collect those

**Comment by dangotbanned** [comment]
I've collected a bunch now that I've seen before and some **new** ones.

I clearly have too much time on my hands ... 😅 

## Seen before
![this narwhal loves you](https://media1.tenor.com/m/X-CluTTK-Y8AAAAC/narwhal-love.gif) ![shaky narwhal](https://steemitimages.com/p/LcTxR7u1XKaa3e4T1EBuBP18JezPvjFFo8gNuE9CiKHBn3j7v3eHTK6u888Hr5fwwnHbD78aU6mDLG7iCb9r2bYBaT6M9hwR3Caw6RbhKfmfbDixNbst2uNbtcktxyUSszUpXPdQFWHgxtMyfX8Mp7ddF?format=match&mode=fit)  <img src="https://media1.tenor.com/m/GpVEolKSa0cAAAAd/unicorn-whale.gif" width=304 alt="very shaky narwhal">

![cat-ptain on a narwhal](https://64.media.tumblr.com/64fb369db15c1b058a5335f8f9ae24ba/tumblr_oql8iwJISo1qak30bo1_500.gifv) ![swing narwhals](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmZkaHJ2N24wdDU4NzcydmQ5bHphNWtmczRzd2FmNTJidHByOXRwayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/mPzDUb0BlwCKsi0vZf/giphy.gif)

![narwhal hugging dolphin](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmZkaHJ2N24wdDU4NzcydmQ5bHphNWtmczRzd2FmNTJidHByOXRwayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/wTJGo31v8rGV2/giphy.gif) ![headspin narwhal](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3bGxnNDdmYWVqZ2Z3anJpd3Bkd2Y5cnlrcjBweHdtZDU4OTV1dmJheiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/c0yYY6zAKejyA6MM0z/giphy.gif)


<img src="https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3azJpZ2hxeDVkeDh2NnlreWpjMWdoZzF3czNocjkzbTV0bnp3c25oNyZlcD12MV9naWZzX3NlYXJjaCZjdD1n/xE2xnByqrmBzP9eyuE/giphy.gif" width=400 alt="squeeze cheeks narwhal"> <img src="https://media1.tenor.com/m/FzNCHOSy5IcAAAAd/swiftmo-twitch.gif" width=600 alt="irl narwhal">

## New to me

![count narwhal](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExaXF1NG1nZWk1cnhvZHoxbzBmY2x5ZWFvcHZzZ2xqdzV3d3FhN211cyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/eEA8wW82Ae7Ep99dCg/giphy.gif) <img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmZkaHJ2N24wdDU4NzcydmQ5bHphNWtmczRzd2FmNTJidHByOXRwayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/R2CWzTrYZt6XcQchOl/giphy.gif" width=500 alt="praise be narwhal">


![spinny hands narwhal](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3d3JvOXR2eDVhNHExOXNzdXFkbTI2aG44OXZ1c3d0bHFsaWdtZHVueCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/7QxG5242BM12HQk1KR/giphy.gif) ![applause narwhal](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmZkaHJ2N24wdDU4NzcydmQ5bHphNWtmczRzd2FmNTJidHByOXRwayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/4LN6iUvWsS1boFTZyl/giphy.gif)


![hungry narwhal](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExdmZkaHJ2N24wdDU4NzcydmQ5bHphNWtmczRzd2FmNTJidHByOXRwayZlcD12MV9naWZzX3NlYXJjaCZjdD1n/QcierOnRPfd29bG1gB/giphy.gif) ![banarwhal](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3bnBpbHZkeHNxcGl5dWQ4aXFqaGJ3NjJxaGM4dGIyaXl2eWduOHE2cCZlcD12MV9naWZzX3NlYXJjaCZjdD1n/BRAov5MaTOTZZnEIVw/giphy.gif)


![welcome fren narwhal](https://media.giphy.com/media/v1.Y2lkPWVjZjA1ZTQ3bGxnNDdmYWVqZ2Z3anJpd3Bkd2Y5cnlrcjBweHdtZDU4OTV1dmJheiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/UokKVBxJlgkutPcK7e/giphy.gif) ![youve got a point narwhal](https://media1.tenor.com/m/pbFSgW2yDjIAAAAC/stickergiant-youve-got-a-point.gif)

**Comment by FBruzzesi** [comment]
Couple more:

![NarrNarrTheNarwhalGIF](https://github.com/user-attachments/assets/03be8bfb-4331-4ddb-ab1f-653dde6d69ea)

![](https://media.tenor.com/eKNMnGRVoZQAAAAC/nautical-narwhals-nft-marketing.gif)
